### ISSUE TITLE
Slash in name

### ISSUE THREAD

**Issue by raicho** [BitBagCommerce/SyliusCmsPlugin#318]
Hello.
Today I caught a mistake...  
```
 The filename and the fallback cannot contain the "/" and "\" characters.
```
I found some solution in my case. In `BitBag\SyliusCmsPlugin\Controller\MediaController::downloadMediaAction` on line 69 i added : 
```php 
$mediaName = str_replace('/', '_', $mediaName);
```

**Comment by bartektw** [comment]
Hi! Can you give more information? How can I reproduce this error?

**Comment by raicho** [comment]
Hello! I'm just adding a file with a name that contains a slash.

**Comment by leszczuu** [comment]
fixed with 3.0.0 release
